site/_collections/algolia.js

```diff
@@ -100,10 +117,25 @@ module.exports = collections => {
       enumerable: true,
     });
 
+    // Add API types to algolia doc to allow fully qualified searches,
+    // like chrome.webRequest.onBeforeSendHeaders instead of just being
+    // able to search onBeforeSendHeaders which also appears in the content
+    if (item.data.api && item.data.chromeApiNamespaces[item.data.api]) {
+      const apiNamespace = item.data.chromeApiNamespaces[item.data.api];
+      const apiNamespacePaths = getChromeApiNamespacePaths(apiNamespace);
+      console.log(item.data.api, apiNamespacePaths);
+      algoliaCollectionItem.apiNamespacePaths = apiNamespacePaths;
+    }
+
     if (item.data.type) {
       algoliaCollectionItem.type = item.data.type;
     }
 
     return algoliaCollectionItem;
   });
 };
+
+module.exports = {
+  getChromeApiNamespacePaths,
```

Comment:
This util function does not seem to have anything in common with algolia. Maybe it's worth moving to _utils, next to group-types? https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/_utils

Reply:
Damn I should trust my intuition 😄 had the same thought but didn't bother as `limitText` was already in there.